src/scripts/OSFramework/OSUI/Constants.ts

```diff
@@ -62,6 +62,8 @@ namespace OSFramework.OSUI.Constants {
 	/* Used to typify the empty string value*/
 	export const EmptyString = '';
 
+	export const FocusTrapIgnoreAttr = 'ignore-focus-trap';
```

Comment:
Because the focus trap implementation is not ready for a use-case like the Balloon, where we have a target that isn't a Block. 

The focus traps add the two spans on the targets parent. In the case of the Balloon, the parent is the OverflowMenu, that also contains the Balloon trigger, which we don't want to be a focusableElement. That's why I created an attribute/flag, so that it can ignore it. Added a comment on that line.

<img width="1629" alt="Screenshot 2023-06-20 at 23 14 08" src="https://github.com/OutSystems/outsystems-ui/assets/32780808/73513a1c-7c84-4b75-9898-9dbaa9731c48">